app/src/main/java/com/teamagam/gimelgimel/app/common/logging/LogWrapperFactory.java

```diff
@@ -0,0 +1,11 @@
+package com.teamagam.gimelgimel.app.common.logging;
+
+/**
+ * A simple factory class to enable future different concrete implementations
+ * to different tags and an easy way to change the LogWrapper implementation throughout the app.
+ */
+public class LogWrapperFactory {
+    public static LogWrapper create(String tag) {
+        return new NativeLogWrapper(tag);
```

Comment:
dont you want to use something like that: 
private Logger(Class clazz, boolean useSimpleName) {
        this.tag = getTag(clazz);
        if (useSimpleName) {
            this.prefixMsg = getPrefixMessage(clazz.getSimpleName());
        } else {
            this.prefixMsg = getPrefixMessage(clazz.getName());
        }
    }

(from: https://github.com/oronno/log4android/blob/master/src/main/java/com/oneous/log4android/Logger.java)